src/components/TodoContainer.js

```diff
@@ -1,10 +1,88 @@
-import PropTypes from  "prop-types"
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+import { useState, useEffect } from "react";
+import styles from "../App.module.css";
 
-function TodoContainer() {}
+function TodoContainer() {
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
 
-TodoContainer.propTypes = {
-    // : PropTypes.func
-  }
-  
+  const fetchData = async () => {
+    const options = {
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
+      },
+    };
+
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/`;
+
+    try {
+      const response = await fetch(url, options);
+
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      }
+
+      const data = await response.json();
+
+      const todos = data.records.map((todo) => ({
+        id: todo.id,
+        title: todo.fields.title,
+      }));
+
+      const mySort = (objectA, objectB) => {
+        if (objectA.title < objectB.title) {
+          return -1;
+        } else if (objectA.title > objectB.title) {
+          return 1;
+        } else {
+          return 0;
+        }
+      };
+
+      setTodoList(todos.sort(mySort));
+      setIsLoading(false);
+    } catch (error) {
+      console.error("Error fetching data:", error.message);
+
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    fetchData(setTodoList, setIsLoading);
+  }, []);
+
+  useEffect(() => {
+    if (!isLoading) {
+      localStorage.setItem("todoList", JSON.stringify(todoList));
+    }
+  }, [todoList, isLoading]);
+
+  const removeTodo = (idToRemove) => {
+    const updatedTodoList = todoList.filter((item) => item.id !== idToRemove);
+    setTodoList(updatedTodoList);
+  };
+
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
```

Comment:
Right now this is only updating the `todoList` in state, you need to add some functionality so that new todos are added to the Airtable db.